README.md

```diff
@@ -47,6 +47,7 @@ You will need a plugin manager to install the plugins and prompts listed on this
 
 - [Official Documentation](http://fishshell.com/docs/current/index.html)
 - [fish cookbook](https://github.com/bucaran/fish-cookbook) - Tips and recipes for fish-shell users
+- [Fish Tutorial](https://github.com/ellerbrock/tutorial-fish-shell-setup-osx) - How to Setup Fish Shell with Fisherman, Powerline Fonts, iTerm2 and Budspencer Theme on OS X.  
```

Comment:
Nice, thanks! I want to merge this, but please change `Fish` to `fish`. No particular reason, other than consistency with the rest of the document. And while you are at it, `Fisherman` to `fisherman` would be nice too.